src/components/views/Login.tsx

```diff
@@ -34,28 +34,28 @@ const Login = () => {
     const [password, setPassword] = useState<string>("");
     const [username, setUsername] = useState<string>("");
 
-    const doLogin = () => {
-        // try {
-        //   // Your login logic goes here
-
-        //   // Placeholder for demonstration
-        //   localStorage.setItem("token", "placeHolder");
-
-        //   // Navigate to the route /game after successful login
-        //   navigate("/game");
-        // } catch (error) {
-        //   alert(`Something went wrong during the login: \n${handleError(error)}`);
-        // }
-        alert("You logged in!")
-        localStorage.setItem("token", "1")
-        navigate("/lobbyoverview")
+    const doLogin = async () => {
+        try {
+            // Your login logic goes here
+            const requestBodyLogin = JSON.stringify({ username, password });
+            const responseLogin = await api.post("/logins", requestBodyLogin);
+            console.log(responseLogin.data);
 
+            // Placeholder for demonstration
+            localStorage.setItem("token", responseLogin.data.token);
+            localStorage.setItem("username", username);
+            // Navigate to the route /game after successful login
+        } catch (error) {
+            alert(`Something went wrong during the login: \n${handleError(error)}`);
+        }
+        alert("You logged in!");
```

Comment:
the user still sees this even in the case when wrong credentials where provided. Only show this pop up when login was successful or even better just navigate when it was successfull and when the credentials where wrong, stay on the login page and let the user retry.